storage/google/cloud/storage/bucket.py

```diff
@@ -253,10 +259,14 @@ def get_blob(self, blob_name, client=None, **kwargs):
         :returns: The blob object if it exists, otherwise None.
         """
         client = self._require_client(client)
-        blob = Blob(bucket=self, name=blob_name, **kwargs)
+        blob = Blob(bucket=self, name=blob_name, encryption_key=encryption_key,
+                    **kwargs)
         try:
+            headers = _get_encryption_headers(encryption_key)
             response = client._connection.api_request(
-                method='GET', path=blob.path, _target_object=blob)
+                method='GET', path=blob.path, _target_object=blob,
+                headers=headers
+            )
```

Comment:
When wrapping the closing paren / bracket to a new line, please add a trailing comma.